changelog/v2.1.md

```diff
@@ -0,0 +1,13 @@
+# Changelog for v2.1
+
+All notable changes to this project for v2.1.Z will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [2.1.0] - 2023-05-03
```

Comment:
Can this be a major version bump instead of a minor? This is technically a breaking change as stuff could be overriden in the cray-service sub chart if desired when deploying the RTS chart (like resource requests, annotations, and other stuff).

Reply:
ok